paparazzi/src/main/java/app/cash/paparazzi/Environment.kt

```diff
@@ -34,12 +32,13 @@ fun detectEnvironment(): Environment {
   val userDir = System.getProperty("user.dir")
   val userHome = System.getProperty("user.home")
   val androidHome = System.getenv("ANDROID_HOME") ?: "$userHome/Library/Android/sdk"
-  val platformDir = Files.list(Paths.get("$androidHome/platforms"))
```

Comment:
This returned Java Streams, I could replace them with Kotlin Sequences but I don't think we would gain anything from it considering the size of collection, so I just used plain collections API.